community-tutorials/ssh-login-with-two-factor-authentication/01-en.md

```diff
@@ -0,0 +1,176 @@
+---
+title: SSH login with Two-Factor Authentication
+description: Learn how to easily secure your SSH login with Two-Factor Authentication (2FA) using Google Authenticator
+updated_at: 2021-11-04
+slug: /ssh-login-with-two-factor-authentication
+author_name: Michael Heinrich
+author_url: -
+author_image: https://www.gravatar.com/avatar/25b58d2cf9f3f0a213259b9c07fd830b
+author_bio: Software Engineer, Maker
+tags: [ssh, security, 2fa, otp, authenticator]
+netcup_product_url: https://www.netcup.de/bestellen/produkt.php?produkt=2000
+language: en
+available_languages: en
+---
+
+# Introduction
+In this tutorial you will learn how to easily secure your SSH login with Two-Factor Authentication (2FA) using `google-authenticator`.
+
+# Requirements
```

Comment:
Which linux distribution are you using in this tutorial?

Reply:
It's for debian and Ubuntu Linux.
I just added it to tags and requirements section.